library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -259,7 +250,7 @@ public boolean handleMessage(@NonNull Message msg) {
             // TODO: handle media scanner timeouts
 
             final boolean isActive;
-            synchronized (downloads) {
+            synchronized (DownloadService.this) {
```

Comment:
Probably worth removing it as it was synchronising access to the `downloads` field which isn't used any more